server/action_creators.js

```diff
@@ -70,6 +72,36 @@ export function addOrder(entries) {
   };
 }
 
+export function changeEntryStatus(orderId, entryId, status) {
```

Comment:
yeah...I had a feeling it would have to be something like this. Entires are embedded inside an order. I'm thinking of redesigning the model so each entry is it's own document...so something like:

Orders collection
 - order id
 - status

Entries collection
 - order id
 - item id
 - status
 - comment

item
 - item id
 - name
 - price

I think this would make querying and updating a little easier. We would just have to pass the object_ids for each document around every where. But i'm not sure how the state would be represented though.